### PR TITLE
Fetch Airdrops

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: airdrop/application/services/user_registration_services.py [hunk 1]
```diff
@@ -9,6 +9,56 @@
...
+            airdrop_window = self.get_user_airdrop_window(
+                airdrop_id, airdrop_window_id
+            )
+
+            if airdrop_window is None:
+                raise Exception(
+                    "Airdrop window is not accepting registration at this moment"
+                )
+
+            is_eligible_user = self.check_user_eligibility(
```
Comment (anandrgitnirman) [inline airdrop/application/services/user_registration_services.py:44]: Why are we hard coding AGIX here ?

File: airdrop/application/services/user_registration_services.py [hunk 2]
```diff
@@ -9,6 +9,56 @@
...
+                    "signature": {"type": "string"},
+                },
+                "required": ["signature", "address", "airdrop_id", "airdrop_window_id"],
+            }
+
+            validate(instance=inputs, schema=schema)
+
+            airdrop_id = inputs["airdrop_id"]
+            airdrop_window_id = inputs["airdrop_window_id"]
+            address = inputs["address"].lower()
```
Comment (anandrgitnirman) [inline airdrop/application/services/user_registration_services.py:32]: checkSum is what we should be using no  ?

File: airdrop/application/services/user_registration_services.py [hunk 3]
```diff
@@ -32,7 +82,7 @@ def register(self, inputs):
             address = inputs["address"].lower()
             signature = inputs["signature"]
 
-            verify_signature(airdrop_id, airdrop_window_id, address, signature)
+            # verify_signature(airdrop_id, airdrop_window_id, address, signature)
```
Comment (anandrgitnirman) [inline airdrop/application/services/user_registration_services.py:85]: this needs to be in scope for demo , why have we commented this  ?

Comment (rajeevbbqq) [reply airdrop/application/services/user_registration_services.py]: Fixed in the latest commit

File: airdrop/application/services/user_registration_services.py (hunk 1 above)
Comment (rajeevbbqq) [reply airdrop/application/services/user_registration_services.py]: Removed hardcoded value AGIX, the actual implementation will update later, for now the eligibility is always `true`

File: airdrop/application/services/user_registration_services.py (hunk 2 above)
Comment (rajeevbbqq) [reply airdrop/application/services/user_registration_services.py:66]: We are doing the checksum conversion explicitly using with utils class

**Review by anandrgitnirman — APPROVED** [review]

**Comment by anandrgitnirman** [comment]
please add test cases , approvig for now

File: airdrop/application/services/user_registration_services.py (hunk 3 above)
Comment (anandrgitnirman) [reply airdrop/application/services/user_registration_services.py]: dont see it yet  ?

**Pull request merged**